src/promptflow/promptflow/_cli/_pf/entry.py

```diff
@@ -103,6 +107,7 @@ def entry(argv):
     Control plane CLI tools for promptflow.
     """
     prog, args = get_parser_args(argv)
+    setup_user_agent_to_operation_context(args.user_agent)
```

Comment:
looks like we have two places setting the user_agent in this file?

Reply:
Yes, one is for CLI itself and this is for ua from CLI params

Reply:
let me put them together in next PR